templates/application.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     targetRevision: {{ $.Values.remoteChart.revision }}
     {{- if $.Values.remoteChart.values }}
     helm:
-      values: {{ nindent 7 (toYaml $.Values.remoteChart.values) }}
+      values: | {{ nindent 7 (toYaml $.Values.remoteChart.values) }}
```

Comment:
could you move the template to a new line?

Reply:
ha! i had changed it following the advice from @Smirl the problem with indenting in a new line is that i have to "break" the yaml flow. to align the underneath YAML properly i'd need to indent from row 0.
If that's not a problem we could leave it like this?

helm lint doesn't dislike it.